app/src/main/res/layout/fragment_sidebar_main.xml

```diff
@@ -236,6 +236,13 @@
                         android:text="@string/fragment_sidebar_main_logout_title"
                         android:textAppearance="?attr/textAppearanceListItemSmall" />
             </LinearLayout>
+
+            <TextView
+                    android:id="@+id/version_info"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
```

Comment:
I think it is better to set layout_height with `?attr/listPreferredItemHeight` like other items if textAppearance is set to `?attr/textAppearanceListItemSmall`, or use custom textAppearance like `@style/TextAppearance.RocketChat.VersionInfo` instead.